gabbi/runner.py

```diff
@@ -241,6 +249,13 @@ def _make_argparser():
         choices=['all', 'body', 'headers'],
         help='Turn on test verbosity for all tests run in this session.'
     )
+    parser.add_argument(
+        '--no-cert-validate',
+        dest='cert_validate',
+        action='store_false',
+        default=True,
```

Comment:
argparse is confusing...

Reply:
Sorry, my comment was unclear. What I was saying was that the combo of:

```
action='store_false',
default=True,
```

was weird.

I have no strong opinion on `--ignore-cert` v `--no-cert-validate`, whichever you prefer.